packages/fiori-elements-writer/src/packageConfig.ts

```diff
@@ -60,9 +63,13 @@ export function getPackageJsonTasks({
     return Object.assign(
```

Comment:
I know it wasn't introduced in your change but why do we use `Object.assign` here? Wouldn't this be better readable with something like 
```
const scripts = {
    start: startCommand,
    'start-local': startLocalCommand
};
if (generateIndex) {
    scripts['start-mock'] = localOnly
        ? `echo \\"${t('info.mockOnlyWarning')}\\"`
        : `fiori run --open "${'index.html'}${searchParam}"`;
}
if (addMock) {
    scripts['start-mock'] = `fiori run --config ./ui5-mock.yaml --open "test/flpSandbox.html${params}"`;
}
...
```

Reply:
I think readability would be more or less the same, but I like that with your suggestion, we don't create new variables if the ```generateIndex``` or the ```addMock``` is false. 

Please review again, I made the requested changes, thanks